.github/CODEOWNERS

```diff
@@ -30,3 +30,15 @@ pkg/postgres/**/*   @stackrox/data-shepherds
 tests/upgrade/*     @stackrox/data-shepherds
 
 operator/**/* @stackrox/draco
+
+# Scanner team's responsibilities include anything related to the scanner itself and scanning utilities
+# such as vulnerability uploading and image integrations.
+/central/scannerdefinitions/                          @stackrox/scanner
```

Comment:
`/central/enrichment`, `/pkg/nodes`, `/pkg/images`?

Reply:
I'll add `pkg/nodes` and `pkg/images/enricher`. I don't think we own `central/enrichment` and `pkg/images` has other things which we don't own

Reply:
FYI, looking at the code it seems like `central/enrichment` has logic that manages integrations and their usage for both images (through deployments) and nodes. Also image metadata caching is initialized there. Also "blame" points out the usual contributors to Scanner (including you). Fine to keep it out, though, lots of intersection. If you want to keep out, we can always add it later if needed. Thanks.

Reply:
Let's keep it out for now, and we can always add it, if we find it is necessary later